cdap-docs/reference-manual/source/http-restful-api/introduction.rst

```diff
@@ -126,10 +126,14 @@ Status Codes
      - The request had a combination of parameters that is not recognized
    * - ``401``
      - ``Unauthorized``
-     - The request did not contain an authentication token
+     - The request did not contain an authentication token; requests can fail due to a
```

Comment:
shouldn't be here. If an operation is unauthorized, it will only throw 403 FORBIDDEN. 401 is thrown only if authentication fails. 403 is when authentication succeeds, but the user is no